### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jugglinmike**
The ecosystem of projects related to the ARIA-AT project is of limited relevance to this proposal's wider audience. Replace it with a listing of known implementations of the protocol.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,5 @@
 # AT Driver API Explainer
 
-**at-driver** &middot; [aria-at-automation-harness](https://github.com/w3c/aria-at-automation-harness) &middot; [aria-at-automation-driver](https://github.com/w3c/aria-at-automation-driver) &middot; [aria-at-automation-results-viewer](https://github.com/w3c/aria-at-automation-results-viewer)
-
-## Authors
-
-* Mike Pennisi
-* Simon Pieters
-
```
Comment (lolaodelola) [inline README.md:9]: Why was this removed?

**Review by lolaodelola — APPROVED** [review]
This LGTM but curious about rm of the authors.

File: README.md (hunk 1 above)
Comment (jugglinmike) [reply README.md:9]: The proposal itself includes attribution.

**Comment by jugglinmike** [comment]
After discovering and correcting a typo on the main branch, I've rebased this branch and force-pushed the result so that the project's automated tests will pass. [The original version of this branch is available here.](https://github.com/w3c/at-driver/compare/main...bocoup:aria-at-automation:update-readme-2-pre-rebase)
